Classes/Domain/Repository/NewsRepository.php

```diff
@@ -251,6 +251,9 @@ protected function createConstraintsFromDemand(QueryInterface $query, DemandInte
         // Id list
         $idList = $demand->getIdList();
         if ($idList) {
+            $query->getQuerySettings()->setRespectStoragePage(false);
+            $query->getQuerySettings()->setRespectSysLanguage(false);
+            $query->getQuerySettings()->setLanguageOverlayMode(true);
```

Comment:
I would also move that out of this important method which is used by **all** actions and make if configurable from the action. what do you think?